packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -3173,6 +3193,7 @@ export interface IDetailsColumnProps extends React.ClassAttributes<DetailsColumn
     updateDragInfo?: (props: {
         itemIndex: number;
     }, event?: MouseEvent) => void;
+    useFastIcons?: boolean;

```

Comment:
does this really need to be an opt in? What is the harm in just optimizing it rather than adding more flags?

Reply:
The fast icons can't be targeted by customizations as part of the theme (they'll pick up the theme colors, but you have to use class names for further customization). So using the fast icons by default would break anyone who might be using customizations on the icons--though there's no way to know if anyone is actually doing that.